release-process-documentation/documentation-guides/update-release-docs-new.md

```diff
@@ -1,88 +1,180 @@
 # Kubernetes.io Docs Release Playbook
 
-author(s): [chenopis@](mailto:chenopis@google.com), stevepe@
+author(s): @chenopis, @steveperry-53
+revisions: @bradamant3, @zacharysarah
 
+This doc shows how to release documentation during a Kubernetes minor version (for example, v1.10) release.
 
-## TL;DR
+## Introduction
 
-This doc contains an outline of tasks to be completed during a Kubernetes.io docs release cycle for a Kubernetes minor release, e.g. v1.9.
+Kubernetes releases minor versions on a quarterly cadence. For example, in 2017, the releases were: v1.6 (Q1/Jan-Mar), v1.7 (Q2/Apr-Jun), v1.8 (Q3/Jul-Sep), and v1.9 (Q4/Oct-Dec).
 
-## Quarterly release cycle
+SIG Docs is part of the Kubernetes release process: its [maintainers](https://github.com/orgs/kubernetes/teams/sig-docs-maintainers) support documentation created/updated for releases and published on the [kubernetes website](https://kubernetes.io/).
 
-The Kubernetes release cycle is on a quarterly cadence. For example, in 2017, the releases are: v1.6 Jan-Mar, v1.7 Apr-Jun, v1.8 Jul-Sep, and v1.9 Oct-Dec. Consequently, [kubernetes.io](https://kubernetes.io/) and its [docs maintainers](https://github.com/orgs/kubernetes/teams/sig-docs-maintainers) are part of that process to support documentation created/updated for those releases.
-
-## Managing the cycle
-
-The designated "release meister" (aka Docs Lead), voluntold from [@kubernetes/sig-docs-maintainers](https://github.com/orgs/kubernetes/teams/sig-docs-maintainers), is responsible for managing the process on the docs side and releasing the documentation on [kubernetes.io](https://kubernetes.io/) when the Kubernetes binary release goes out.
+The designated "release meister" (aka Docs Lead), voluntold from [@kubernetes/sig-docs-maintainers](https://github.com/orgs/kubernetes/teams/sig-docs-maintainers), is responsible for managing the docs release  for new and updated documentation [kubernetes.io](https://kubernetes.io/) during quarterly releases.
 
 ### Relevant links
 
 *   k8s.io website - [https://kubernetes.io/](https://kubernetes.io/)
-*   k8s.io (docs) GitHub repo - [kubernetes/kubernetes.github.io](https://github.com/kubernetes/kubernetes.github.io)
-*   k8s.io [pull request (PR) queue](https://github.com/kubernetes/kubernetes.github.io/pulls) 
-*   OSS Kubernetes features tracking spreadsheet
-    *   [Kubernetes Features OSS tracking board (1.7 release)](https://docs.google.com/spreadsheets/d/1IJSTd3MHorwUt8i492GQaKKuAFsZppauT4v1LJ91WHY/edit?usp=sharing)
-    *   [Kubernetes Features OSS tracking board (1.8 release)](https://docs.google.com/spreadsheets/d/1AFksRDgAt6BGA3OjRNIiO3IyKmA-GU7CXaxbihy48ns/edit#gid=0)
-    *   Not sure whether there will be a tracking spreadsheet for 1.9
+*   k8s.io (docs) GitHub repo - [kubernetes/website](https://github.com/kubernetes/website)
```

Comment:
Would it be better to use kubernetes.io instead of k8s.io as the "official website" in all of these list items?